<launch>
    <node name="registration_node" pkg="nrs_ver2" type="registration.py" output="screen" />
</launch>
